eagleproject/core/coingecko.py

```diff
@@ -5,10 +5,12 @@
 log = get_logger(__name__)
 
 COINGECKO_ENDPOINT = "https://api.coingecko.com/api/v3"
+DEFILLAMA_ENDPOINT = "https://stablecoins.llama.fi"
```

Comment:
Would suggest to change the name of this file to something else, since it no longer has only Coingecko API code

Reply:
separated the files